### ISSUE TITLE
--storage_directory param for --export shouldn't be optional

### ISSUE THREAD

**Issue by yanokwa** [opendatakit/briefcase#452]
#### Software versions 
Briefcase v1.x.x, Java v1.x.x, operating system, Aggregate v1.x.x, Collect v1.x.x...

#### Problem description
According to Briefcase's help, the storage directory is optional.
```
Params for -e operation:
...
  -sd,--storage_directory <arg>       Briefcase storage directory
  (optionally)
```
I don't think it is.

**Comment by ggalmazor** [comment]
OK, that `(optionally)` is there to mean that the following params are optional (not the one on the line before that). To make things less ambiguous, we could change this:

```
Params for -e operation:
  -ed,--export_directory <arg>        Export directory
  -f,--export_filename <arg>          Filename for export operation
  -id,--form_id <arg>                 Form ID
  -sd,--storage_directory <arg>       Briefcase storage directory
  (optionally)
  -em,--exclude_media_export          Exclude media in export
  -end,--export_end_date <arg>        Export end date (yyyy-MM-dd or yyyy/MM/dd)
  -oc,--overwrite_csv_export          Overwrite files during export
  -pf,--pem_file <arg>                PEM file for form decryption
  -start,--export_start_date <arg>    Export start date (yyyy-MM-dd or yyyy/MM/dd)
```

Into this:

```
Params for -e operation:
  -ed,--export_directory <arg>        Export directory
  -f,--export_filename <arg>          Filename for export operation
  -id,--form_id <arg>                 Form ID
  -sd,--storage_directory <arg>       Briefcase storage directory
Optional params for -e operation:
  -em,--exclude_media_export          Exclude media in export
  -end,--export_end_date <arg>        Export end date (yyyy-MM-dd or yyyy/MM/dd)
  -oc,--overwrite_csv_export          Overwrite files during export
  -pf,--pem_file <arg>                PEM file for form decryption
  -start,--export_start_date <arg>    Export start date (yyyy-MM-dd or yyyy/MM/dd)
```

**Comment by yanokwa** [comment]
Closed by #441
